packages/badge/src/index.vue

```diff
@@ -12,6 +12,7 @@
             'is-dot': isDot
           }
         ]"
+        @click.stop="handleClick"
```

Comment:
BTW, you can just change the code here as
```suggestion
@click.stop="$emit('click')"
```